SU-HSDO/stanford_mrc
# MRCD8-245 added latest stanford_basic which includes site title fix

# READY FOR REVIEW

# Summary
- In stanford_basic, I updated the brandbar template and styles
- This PR includes the latest stanford_basic in stanford_mrc

# Needed By (Date)
- End of sprint (3/1)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify that the site title is responsive
3. Slogan should be dropped below Stanford wording and logo image should not display on mobile (481px and below)

# Affected Projects or Products
- stanford_basic
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-245
- Commits: https://github.com/SU-SWS/stanford_basic/commit/5e70c3585d2e220be494e56712ac8ef37002a7bc and https://github.com/SU-SWS/stanford_basic/commit/754b3a887cba7896305064da7ace6b0ccdd30114

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
